src/nvector/manyvector/nvector_manyvector.c

```diff
@@ -220,37 +220,41 @@ N_Vector N_VNew_MPIManyVector(sunindextype num_subvectors,
 {
   sunindextype i;
   sunbooleantype nocommfound;
-  void* tmpcomm;
-  MPI_Comm comm, *vcomm;
+
+  MPI_Comm comm, vcomm;
   int retval, comparison;
   N_Vector v = NULL;
 
   /* Check that all subvectors have identical MPI communicators (if present) */
   nocommfound = SUNTRUE;
-  comm = MPI_COMM_NULL;
+  vcomm = MPI_COMM_NULL;
```

Comment:
```suggestion
  vcomm = SUN_COMM_NULL;
```

Reply:
No need to use `SUN_COMM_NULL` since this is only compiled if we have MPI. As I said in the other comments, I slightly prefer using `MPI_Comm` and `MPI_COMM_NULL` in code that only compiles with MPI enabled but if others feel differently there is not really a downside to changing it.